### DISCUSSION TITLE
Windows Securly (possibly SSL certificate)

### DISCUSSION THREAD

**Discussion by PSYclownYT** [3kh0/ext-remover#960] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
Is there a (free) bypass that I can use to disable securely on windows? (I am not willing to buy kon - boot because its like $20) I am on windows 10 for education.

**Comment by ghost** [comment]
i know
virtualbox

**Reply by PSYclownYT** [reply]
Oh yeah we also cant install shit sorry for not saying that

**Comment by S-PScripts** [comment]
closed unless PSYclownYT wants to reopen
